### PR TITLE
[POLICIES.md] Initial commit

### PR DESCRIPTION
**Pull request by joshlf**
Document our policies on soundness and MSRV.

### CODE REVIEW

File: POLICIES.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# Zerocopy's Policies
+
+## Soundness
+
+Zerocopy is expressly designed for use in security-critical contexts. It is used
+in hardware security firmware, cryptographic implementations, hypervisors, and
+more. We understand that software in these contexts has a very high bar for
+correctness, and we take our responsibility to meet that bar very seriously.
```
Comment (jswrenn) [inline POLICIES.md:8]:
```suggestion
correctness, and we take our responsibility to exceed that bar very seriously.
```

File: POLICIES.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+Each non-test `unsafe` block must be annotated with a "safety comment" which
+provides a rationale for its soundness. In order to ensure that our soundness is
+forwards-compatible, safety comments must satisfy the following criteria:
+- Safety comments must constitute a (possibly informal) proof that all of Rust's
+  soundness rules are upheld.
+- Safety comments must not rely for their correctness on any statements which do
+  not appear in the stable versions of the [Rust Reference] or standard library
+  documentation (ie, the docs for [core], [alloc], and [std]); arguments which
+  rely on text from the beta or nightly versions of these documents are not
+  considered complete.
```
Comment (jswrenn) [inline POLICIES.md:35]: The negative framing of this point ("must **not** rely") is somewhat confusing.

File: README.md
Comment (jswrenn) [inline README.md:1]: Did you mean to also modify `lib.rs`?

**Review by jswrenn — CHANGES_REQUESTED** [review]

File: POLICIES.md (hunk 1 above)
Comment (joshlf) [reply POLICIES.md:8]: Discussed offline; going to leave as-is.

**Review by jswrenn — DISMISSED** [review]

**Review by jswrenn — APPROVED** [review]

**Pull request merged**